scripts/format.sh

```diff
@@ -10,8 +10,9 @@
 set -o pipefail
 cd "$(dirname "$0")/.."
 
-bazel build @rules_scala_annex//rules/scalafmt
-scalafmtbin="bazel-bin/external/rules_scala_annex/rules/scalafmt/scalafmt-bin"
+scalafmtbin="$(mktemp)"
```

Comment:
I don't think this will work if the default temp directory (/tmp on linux) is mounted as noexec, which is fairly common. 

Some possible workarounds:

- If the script is a shell script, always invoke with `bash` instead of calling it directly
- Write to a well-known path in the current directory
- Use `--tmpdir` to `mktemp` to have it create the file in a directory that is more likely to allow execution, like the current directory

Reply:
Good point; I hadn't thought of that. I like the first alternative you listed the most, since it doesn't leave the possibility of a lingering script in the working directory. I'll go with that.